CHANGELOG.md

```diff
@@ -1,127 +1,32 @@
-| `SearchViewModel.defaultSymbol` | Use [SearchViewModel.defaultSymbols](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Search-SearchViewModel.html#defaultSymbols) instead | 4.22 |
-| `StreamLayerView.effect` | Use [StreamLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-StreamLayerView.html#featureEffect) instead | 4.22 |
-| `WFSLayerView.effect` | Use [WFSLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-WFSLayerView.html#featureEffect) instead | 4.22 |
+| `esri/widgets/Slice/SlicePlane` | Use [esri/analysis/SlicePlane](https://developers.arcgis.com/javascript/latest/api-reference/esri-analysis-SlicePlane.html) instead | 4.23 |
 
 Please refer to the [Breaking changes](https://developers.arcgis.com/javascript/latest/breaking-changes/) guide topic for a complete list of breaking changes across all releases of the 4.x.
 
 ## Bug fixes and enhancements
 
-- BUG-000135050: Fixed an issue where Popups would not display attachments when sublayers on a MapImageLayers had AttachmentsContent configured. This only applies to map services on ArcGIS Enterprise version 10.8.1 and above.
-- BUG-000146636: Fixed an issue where using the Zoom To action in popups would zoom passed the selected geometry's extent.
-- BUG-000153564: Fixed an issue where VectorTileLayer intermittently stopped drawing some tiles at certain zoom levels.
-- BUG-000153928: Fixed an issue where Area Measurement failed if the second point was added via a double-click instead of single click.
-- BUG-000154321: Fixed an issue where LineOfSightViewModel `observer` and `targets` returned z-values in varying elevation modes. Now all z-positions are returned as absolute values.
-- BUG-000154436: Fixed an issue within the FeatureTable widget where hiding a child column contained within a grouped column would not work correctly. This has been fixed and these contained columns can be hidden when programatically setting their visibility to `false`.
-- Esri Community - 1241411: Fixed an issue where an AbortError thrown when deleting a layer after changing its definitionExpression.
-- Fixed an issue where MediaContent charts in a popup displaying information from a relationship were only showing one related feature.
-- Fixed an issue where two `queryAttachments` requests were sent to the service when selecting a feature that had AttachmentsContent in its Popup.
-- ENH-000152699: Added support for [ImageryTileLayer](/api-reference/esri-layers-ImageryTileLayer.html) referencing a tiled image services with LODs that are not power of two.
-- ENH-000155208: Enhanced TileLayer to highlight the selected feature when displaying its Popup.
+* TBD
 
 ## Deprecations
 
```

Comment:
should we add the View popup deprecation here??

Reply:
Good call, adding it.

Reply:
Updated, thanks.